### DISCUSSION TITLE
Does it support File manager transfers?

### DISCUSSION THREAD

**Discussion by necros2k7** [bk138/droidVNC-NG#210] [Q&A] [answered]
@bk138 , @eisaev , @Poussinou If yes what win client does support it? I tried to connect to Android 8 nonroot with TightVNC - file transfer mode greyed out

**Comment by bk138** [comment, marked as answer]
Yes, but only for older TightVNC versions, see https://github.com/bk138/droidVNC-NG?tab=readme-ov-file#features
